cmds/core/kexec/kexec_linux.go

```diff
@@ -48,6 +49,7 @@ type options struct {
 
 func registerFlags() *options {
 	o := &options{}
+	flag.IntVarP(&o.syscall, "syscall", "", 1, "Select which syscall to use, 0 for kexec_load syscall, and 1 for file load. Default: 1")
```

Comment:
can we make this a string? It's just as easy to use a string. The options would then be
syscall
file

Reply:
Done.

making the option vals as: (`kexecload`, `fileload`), as for ("syscall",  "file"), it seems a bit weird when doing `--syscall syscall`